docs/changelog.md

```diff
@@ -187,9 +187,9 @@ modular update mojo
   v.append("Bob")
   v.append("Charlie")
   for x in v:
-    x[] = str("Hello, ") + x[]
+      x[] = str("Hello, ") + x[]
```

Comment:
I actually prefer 2-space indentation, but my preference hardly matters. This way, it's more consistent.